docker-compose.test.yaml

```diff
@@ -6,4 +6,4 @@ version: '2.4'
 services:
     postgres:
         environment:
-            POSTGRES_DB: boost-test
+            POSTGRES_DB: einstore
```

Comment:
You can delete this file altogether, and remove `-f docker-compose.yaml -f docker-compose.test.yaml` from Makefile